index.md

```diff
@@ -51,7 +53,7 @@ direction: rtl
 </div>
 </details>
 
-<details class="links-section">
+<details class="links-section"  id="section3">
```

Comment:
Maybe instead put the id=sectionName? it will look nicer when sharing the link?

Reply:
can u chack it now?